modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/discovery/DataDiscovery.scala

```diff
@@ -41,7 +46,9 @@ case class DataDiscovery(
   shreddedTypes: List[ShreddedType],
   compression: Compression,
   typesInfo: TypesInfo,
-  columns: List[String]
+  wideColumns: List[String],
+  shredModels: Map[SchemaKey, MergeRedshiftSchemasResult],
+  disableMigration: List[SchemaCriterion]
```

Comment:
Here you copy `disableMigration` from the config into the `DataDiscovery` case class.  You do this so that you can use it in the `Migration` object.

Did you consider instead simply passing the relevant config down into `Migration` object but without adding it to the `DataDiscovery` class?

I don't know if that would look neater or not.  Ultimately we should do whichever implementation looks neatest.

I suggest the alternative because `DataDiscovery` kinda represents information we discover dynamically from the message queue or from Iglu.  And `disableMigration` does not seem to fit that description.

But I will leave it to your judgement whether your implementation is neater than my suggestion.  I haven't thought through the details or the impact on the code.

Reply:
`getLoadStatements` also use `disableMigration` to decide which table name should be used but I agree with your sentiment, I just pushed a new commit